index.html

```diff
@@ -1,57 +1,226 @@
+            type="email"
+            name="email"
+            id="email"
+            class="form-control"
+            placeholder="Enter your Email"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label id="number-label" for="number"
+            >Age<span class="clue">(optional)</span></label
+          >
+          <input
+            type="number"
+            name="age"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="Age"
+          />
+        </div>
+        <div class="form-group">
+          <p>Which option best describes your current role?</p>
+          <select id="dropdown" name="role" class="form-control" required>
+            <option disabled selected value>Select current role</option>
+            <option value="student">Student</option>
+            <option value="job">Full Time Job</option>
+            <option value="unemployed">Unemployed</option>
+            <option value="preferNo">Prefer not to say</option>
+            <option value="other">Other</option>
+          </select>
+        </div>
+    
+        <div class="form-group">
+          <p>Would you recommend our meditation app to a friend?</p>
+          <label>
+            <input
+              name="user-recommend"
+              value="definitely"
+              type="radio"
+              class="input-radio"
+              checked
+            />Definitely</label
+          >
+          <label>
+            <input
+              name="user-recommend"
+              value="maybe"
+              type="radio"
+              class="input-radio"
+            />Maybe</label
+          >
+    
+          <label
+            ><input
+              name="user-recommend"
+              value="not-sure"
+              type="radio"
+              class="input-radio"
+            />Not sure</label
+          >
+        </div>
+    
+        <div class="form-group">
+          <p>
+            What is your favorite meditation app?
+          </p>
+          <select id="most-like" name="mostLike" class="form-control" required>
+            <option disabled selected value>choose</option>
+            <option value="Amaha">Amaha</option>
+            <option value="Balance">Balance</option>
+            <option value="Being">Being</option>
+            <option value="Calm">Calm</option>
+            <option value="Headspace">Headspace</option>
+            <option value="Zen">Zen</option>
+          </select>
+        </div>
+    
+        <div class="form-group">
+          <p>
+            What would you like to see improved?
+            <span class="clue">(Check all that apply)</span>
+          </p>
+    
+          <label
+            ><input
+              name="prefer"
+              value="goal-tracker"
+              type="checkbox"
+              class="input-checkbox"
+            />Goal tracker</label
+          >
+          <label>
```

Comment:
Kindly close the tags within one line if possible to enhance a good visual appeal when viewers are going through your code. Example: `<label id="id-name" for="purpose">GPA<span class="class-name">4.0</span></label>`